eng/common/testproxy/transition-scripts/generate-assets-json.ps1

```diff
@@ -348,9 +355,29 @@ $language = Get-Repo-Language
 # in the path and that we're able to map the language's recording
 # directories
 if ($InitialPush) {
-  Test-Exe-In-Path -ExeToLookFor $TestProxyExe
+  $proxyPresent = Test-Exe-In-Path -ExeToLookFor $TestProxyExe -ExitOnError $false
+
+  # try to fall back 
+  if (-not $proxyPresent) {
+    $StandaloneTestProxyExe = "Azure.Sdk.Tools.TestProxy"
```

Comment:
I think I missed this. Why is this the standalone tool name and not `test-proxy`? It strikes me as pretty odd that a cli tool would be named like this. Is the dotnet tool install also using this name or still `test-proxy` (and if the latter I think we want our tool names to be consistent from the docs).

Reply:
No, when you produce a standalone executable it's named for the namespace of the project.

I can change it, but then it would make the produced dll `test-proxy.dll`, which I do not want. When I last dug into this to produce the signed standalone executables, that was all that I could find.

Reply:
@benbp I'm going to merge this to unblock the java folks, then I'm going to make the standalone just named `test-proxy`.

The latter one is going to be a fun update to `azure-sdk-build-tools` (there is a reason I didn't do it at first 😂) but you're absolutely right about your original point so...